File: Archive/ASP.NET/365ThreeSixty_v2/365ThreeSixty_v2/vote.vb
Public Class vote
    Private pReviewerRef As Integer
    Private pRecipientRef As Integer
    Private pRawScore As Integer
    Private pWeightedScore As Double
    Private pComment As String
    Private pCommentScore As Double
    Private pTierFactor As Double
    Private pReviewerFactor As Double
    Private pReviewerScore As Double
    Private pRecipientFactor As Double
    Private pRecipientScore As Double

    Public Property reviewer() As Integer
        Get
            Return pReviewerRef
        End Get
        Set(value As Integer)
            pReviewerRef = value
        End Set
    End Property

    Public Property recipient() As Integer
        Get
            Return pRecipientRef
        End Get
        Set(value As Integer)
            pRecipientRef = value
        End Set
    End Property

    Public Property rawScore() As Integer
        Get
            Return pRawScore
        End Get
        Set(value As Integer)
            pRawScore = value
        End Set
    End Property

    Public Property weightedScore() As Double
        Get
            Return pWeightedScore
        End Get
        Set(value As Double)
            pWeightedScore = value
        End Set
    End Property

    Public Property tierFactor() As Double
        Get
            Return pTierFactor
        End Get
        Set(value As Double)
            pTierFactor = value
        End Set
    End Property

    Public Property reviewerFactor() As Double
        Get
            Return pReviewerFactor
        End Get
        Set(value As Double)
            pReviewerFactor = value
        End Set
    End Property
    Public Property reviewerScore() As Double
        Get
            Return pReviewerScore
        End Get
        Set(value As Double)
            pReviewerScore = value
        End Set
    End Property
    Public Property RecipientFactor() As Double
        Get
            Return pRecipientFactor
        End Get
        Set(value As Double)
            pRecipientFactor = value
        End Set
    End Property
    Public Property recipientScore() As Double
        Get
            Return pRecipientScore
        End Get
        Set(value As Double)
            pRecipientScore = value
        End Set
    End Property

    Public Property comment() As String
        Get
            Return pComment
        End Get
        Set(value As String)
            pComment = value
        End Set
    End Property

    Public Property commentScore As Double
        Get
            Return pCommentScore
        End Get
        Set(value As Double)
            pCommentScore = value
        End Set
    End Property
End Class

File: 365ThreeSixtyAPI/365ThreeSixtyAPI/Models/vote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace _365ThreeSixtyAPI.Models
{
    public class vote
    {
    public int id { get; set; }
    public int reviewerRef { get; set; }
    public int recipientRef { get; set; }
    public int rawScore { get; set; }
    public double weightedScore { get; set; }
    public string comment { get; set; }
    public double commentFactor { get; set; }
    public double commentScore { get; set; }
    public double tierFactor { get; set; }
    public double tierScore { get; set; }
    public double reviewerFactor { get; set; }
    public double reviewerScore { get; set; }
    public double recipientFactor { get; set; }
    public double recipientScore { get; set; }
    public DateTime voteSubmittedAt { get; set; }
    public string userAccountId { get; set; }
    }
}